charts/asset-manager/templates/clamav-db-persistentvolumeclaim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
```

Comment:
I'm guessing the EFS CSI driver takes care of the PersistentVolume side of things and that's why we don't define a PersistentVolume ourselves? Is that right?

Reply:
yep